ts/sagas/contentLoaders.ts

```diff
@@ -53,48 +53,59 @@ export function* watchContentServiceLoadSaga(): Iterator<Effect> {
     action: ActionType<typeof contentServiceLoad["request"]>
   ) {
     const serviceId = action.payload;
+    try {
```

Comment:
We can test further this part together with the changes being introduced within https://github.com/teamdigitale/io-app/pull/1352